torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -704,7 +817,7 @@ void propagateDequantize(Value* output, const std::vector<Value*> inputs) {
   insertDeQuantForAllUse(graph, output, output);
 }
 
-void propagateQuantizationOps(Block* block) {
+void InsertQuantDeQuantHelper::propagateQuantizationOps(Block* block) {
```

Comment:
Can you add a comment stating for which ops we do propagation with this function?

Reply:
added in declaration